docs/source/howto/workflows.rst

```diff
@@ -20,6 +20,172 @@ Running a predefined workflow
+This can of course be achieved by importing the function or class from its respective module:
+
+.. code-block:: python
+
+    from aiida.workflows.arithmetic.add_multiply import add_and_multiply              # Import the add_and_multiply work function
+    from aiida.workflows.arithmetic.multiply_add import MultiplyAddWorkChain       # Import the MultiplyAddWorkChain WorkChain
+
+However, a more robust way of loading a workflow is using the ``WorkflowFactory``, for example:
+
+.. code-block:: python
+
+    from aiida.plugins import WorkflowFactory
+    add_and_multiply = WorkflowFactory('arithmetic.add_multiply')
+    MultiplyAndAddWorkChain = WorkflowFactory('arithmetic.multiply_add')
+
+The input argument of the ``WorkflowFactory`` is a string called the *entry point*.
+For each workflow, the entry point usually starts with with the name of the plugin package (``arithmetic``), followed by a dot and the string identifying the workflow (``multiplyadd``).
+Using the ``WorkflowFactory`` has the advantage that the entry point will not change when the packages or plugins are reorganised, and hence your code is less likely to break when updating AiiDA or the plugin that supplies the workflow.
+
+The list of installed plugins can be easily accessed via the verdi CLI:
+
+.. code-block:: console
+
+    $ verdi plugin list
+
+To see the list of ``WorkChain`` entry points, simply use:
+
+.. code-block:: console
+
+    $ verdi plugin list aiida.workflows
+
+By further specifying the entry point of the workflow, you can see its description, inputs, outputs and exit codes:
+
+.. code-block:: console
+
+    $ verdi plugin list aiida.workflows arithmetic.multiplyadd
+
+.. note::
+
+    Using the ``WorkflowFactory`` is a good paradigm to get used to as the entry point names are unique identifiers that are supported in many parts of AiiDA's API, such as the ``verdi plugin list`` command shown above.
+    The complete list of available plugins for AiiDA can be found at the `AiiDA plugin registry <https://aiidateam.github.io/aiida-registry/>`_.
+    If you want to write your own workflow, please see the section on :ref:`how to write your own workflows<how-to:workflows:write>`.
+
+Work functions
+--------------
+
+Running a work function is as simple as calling a typical Python function: simply call it with the required input arguments:
+
+.. code-block:: python
+
+    add_and_multiply = WorkflowFactory('arithmetic.add_multiply')
+    Int = DataFactory('int')
+
+    result = add_and_multiply(Int(2), Int(3), Int(5))
+
+Here, the ``add_and_multiply`` work function returns the output ``Int`` node and we assign it to the variable ``result``.
+Note that the input arguments of a work function must be an instance of ``Data`` node, or any of its subclasses.
+Just calling the ``add_and_multiply`` function with regular integers will result in a ``ValueError``, as these cannot be stored in the provenance.
```

Comment:
```suggestion
Just calling the ``add_and_multiply`` function with regular integers will result in a ``ValueError``, as these cannot be stored in the provenance graph.
```

Correct me if I'm wrong, but the term [*provenance*](https://en.wikipedia.org/wiki/Provenance) for me basically translates to "origin" or "history". I don't think that you can "store anything within the provenance".